conan_server/Dockerfile

```diff
@@ -13,8 +13,10 @@ RUN apk add --no-cache gcc libc-dev \
     && cd conan-${CONAN_VERSION} \
     && pip3 install -r conans/requirements.txt \
     && pip3 install -r conans/requirements_server.txt \
+    && pip3 install . \
```

Comment:
Why this?

Reply:
the problem occurs because gnunicorn runs 4 workers at startup and it cause a conflict because all the workers try to create ~/.conan_server/data at same time.

I installed conan to be to execute conan_server --migrate and populate ~/.conan_server/data.

The point is that we need to fix the race condition when there is no data folder there. Any better idea?

Reply:
why not just create a folder then (`mkdir`)?

Reply:
because there are also `server.conf` and `version.txt` at same level.

Reply:
Now it won't install Conan, but --migrate is required just to populate the conan_server folder.